rpcs3/Emu/RSX/VK/VKHelpers.cpp

```diff
@@ -139,6 +139,9 @@ namespace vk
 			rsx_log.error("Dozen is currently unsupported. How did you even get this to run outside windows?");
 #endif
 			break;
+		case driver_vendor::HONEYKRISP:
+			// Needs more testing
+			break;
```

Comment:
missing break below

Reply:
You mean the default tail block? That doesn't need a break statement.

Reply:
That's what we always do...
Weird reason to break code style and best practises now

Reply:
This isn't new code.